src/ActivitiesSDKAdapter.js

```diff
@@ -224,24 +212,19 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
    */
   // eslint-disable-next-line class-methods-use-this
   getAdaptiveCard(activity) {
-    const hasCard = this.hasAdaptiveCard(activity);
-
-    return hasCard ? activity.attachments[0].content : undefined;
+    return activity.card;
   }
 
   /**
-   * A function that attaches an adaptive card to an Activity object.
+   * A function that adds an adaptive card to an Activity object.
    *
    * @param {Activity} activity  The activity to post
-   * @param {object} card  The card attachment
+   * @param {object} card  The adaptive card
    */
   // eslint-disable-next-line class-methods-use-this
   attachAdaptiveCard(activity, card) {
     const mutableActivity = activity;
```

Comment:
This doesn't really make it mutable, it's just a reference to the same parameter. You need to make a copy of the parameter

Reply:
I know, we want to change the values in this object, not in a copy. The reason is that before posting an activity we will attach the card in this way:
`const activityToPost = {...};
const cardToPost = {...};

attachAdaptiveCard(activityToPost, cardToPost);
postActivity(activityToPost);`